### PR TITLE
Fellowships assets

### PR DESCRIPTION
**Pull request by mmmavis**
Latest Sketch file: https://drive.google.com/drive/folders/13B0EMMavSNsibacHIpJd1sMi9TvslR7o (mobile comps are outdated so just refer to desktop mockups)

- Added illustrations to the following pages
  - /fellowships
  - /fellowships/science
  - /fellowships/open-web
  - /fellowships/tech-policy
  - /fellowships/media
  - /fellowships/support
- Ran `npm run optimize` to optimize image assets (❗️❗️❗️this optimized a ton of existing images in the repo. i hope image optimization would not break anything)

### CODE REVIEW

File: source/images/burst1.svg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<svg width="178" height="157" viewBox="0 0 178 157" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (mmmavis) [inline source/images/burst1.svg]: @gvn is `viewBox` safe to remove? 👀

Comment (gvn) [reply source/images/burst1.svg]: I'm not sure...I don't usually hand-optimize SVG files.

**Comment by gvn** [comment]
Also, FYI: You can just run `npm run optimize:svg` (or :png or :svg) to avoid optimizing every image type.

**Comment by mmmavis** [comment]
@gvn yea I'll need to rebase the main `fellowships` branch once your refactoring work in `master` is done.

**Comment by gvn** [comment]
@mmmavis Once this lands you should do a rebase:

https://github.com/mozilla/foundation.mozilla.org/pull/1067

**Comment by gvn** [comment]
Actually...it should be safe for you to rebase now since you're not touching the `campaign` or `landingpage` templates.

`base-compiled` is in master now

**Comment by mmmavis** [comment]
@gvn I'll merge this PR into `fellowships` branch first. I'll file a separate PR to do the `master` <-> `fellowships` rebase after.

**Pull request merged**